src/lib/tcti.c

```diff
@@ -110,26 +141,38 @@ init_tcti_device(void)
 TSS2_TCTI_CONTEXT *
 init_tcti_socket(void)
 {
+#ifndef TSS2_LEGACY_V1
+	const char *cfg = "host=127.0.0.1,port=2321";
```

Comment:
Is there a macro defining the port number and local address in a tss2 header?

Reply:
I was trying to find the macro, but unfornately did find it. 

Some sample code in tpm2-tss are all hard-coded:

    expect_value(__wrap_Tss2_Tcti_Mssim_Init, tctiContext, NULL);
    expect_any(__wrap_Tss2_Tcti_Mssim_Init, size);
    expect_string(__wrap_Tss2_Tcti_Mssim_Init, config, "host=localhost,port=2321");
    will_return(__wrap_Tss2_Tcti_Mssim_Init, lsize);
    will_return(__wrap_Tss2_Tcti_Mssim_Init, TSS2_RC_SUCCESS);

    expect_any(__wrap_Tss2_Tcti_Mssim_Init, tctiContext);
    expect_memory(__wrap_Tss2_Tcti_Mssim_Init, size, &lsize, sizeof(lsize));
    expect_string(__wrap_Tss2_Tcti_Mssim_Init, config, "host=localhost,port=2321");
    will_return(__wrap_Tss2_Tcti_Mssim_Init, lsize);